examples/ucbadmit.py

```diff
@@ -94,11 +106,24 @@ def main(args):
     dept, male, applications, admit = fetch_train()
     rng_key, rng_key_predict = random.split(random.PRNGKey(1))
     zs = run_inference(dept, male, applications, admit, rng_key, args)
-    rng_keys = random.split(rng_key_predict, args.num_samples * args.num_chains)
-    pred_probs = vmap(lambda z, rng_key: predict(dept, male, applications, z, rng_key))(zs, rng_keys)
+    pred_probs = Predictive(glmm, zs).get_samples(rng_key_predict, dept, male, applications)['probs']
     header = '=' * 30 + 'glmm - TRAIN' + '=' * 30
     print_results(header, pred_probs, dept, male, admit / applications)
 
+    # make plots
+    fig, ax = plt.subplots(1, 1)
+
+    ax.plot(range(1, 13), admit / applications, "o", ms=7, label="actual rate")
+    ax.errorbar(range(1, 13), np.mean(pred_probs, 0), np.std(pred_probs, 0),
+                fmt="o", c="k", mfc="none", ms=7, elinewidth=1, label=r"mean $\pm$ std")
+    ax.plot(range(1, 13), np.percentile(pred_probs, 5, 0), "k+")
+    ax.plot(range(1, 13), np.percentile(pred_probs, 95, 0), "k+")
+    ax.set(xlabel="cases", ylabel="admit rate", title="Posterior Predictive Check with 90% CI")
+    ax.legend()
+
+    plt.savefig("ucbadmit_plot.pdf")
+    plt.tight_layout()
```

Comment:
Does this have any impact if it is after the `savefig`?

Reply:
Yes, this removes borders of the figure (hence the thumbnails will be a bit bigger). `savefig` does not affect the output.